src/main/java/com/google/sps/utils/data/Animal.java

```diff
@@ -0,0 +1,205 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Collection;
+import java.util.HashSet;
+
+public class Animal {
+    
+    // Cross-referenced
+    private String common_name;
+    private String binomial_name;
+
+    // Provided by Wikipedia
+    private int population;
+    private String status;
+
+    // Provided by IUCN API
+    private int id;
+    private String kingdom;
+    private String phylum;
+    private String class;
+    private String order;
+    private String family;
+    private String genus;
+    private String population_trend;
```

Comment:
Hmm population trend could be a boolean or an int? We only need decreasing or increasing so we could rename it to be something like "isPopulationDecreasing" and mark it true or false.

Reply:
Done.